OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -64,8 +68,9 @@ static object LoadShape(MiniYaml yaml)
 
 	public class HitShape : ConditionalTrait<HitShapeInfo>, ITargetablePositions
 	{
-		BodyOrientation orientation;
+		protected BodyOrientation orientation;
```

Comment:
Does this still need to be `protected`?

Reply:
Nah, this and the other ones basically are copy-pasta leftovers from back when downstream was using a `TurretHitShape`. I'll remove them.